### DISCUSSION TITLE
Maximum call stack size exceeded

### DISCUSSION THREAD

**Discussion by GabrielAllbreaker** [google/model-viewer#4161] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/110112903/225101378-905a87d7-accb-48a3-8ac4-a03f8c204dcf.png)

Hello!, I'm trying to see a glb model with animation but this don't work, i tried in a web page from github and a local page with xampp but doesn't work in different devices, I attach an image of the error that I saw in the console

**Comment by elalish** [comment, marked as answer]
Indeed, we just ran into this internally as well. It was a regression we picked up from three.js, which has already been fixed upstream: https://github.com/mrdoob/three.js/pull/25377

We just need to push a new release with the fix. Stay tuned.

**Comment by GabrielAllbreaker** [comment]
thanks I'll be attentive

**Comment by quangtri3111996** [comment]
Hello @GabrielAllbreaker and @elalish Any update to fix it ?

**Reply by diegoteran** [reply]
Are you still seeing this issue @quangtri3111996? It seems both three.js and model-viewer had new releases after the regression was fixed.

**Reply by quangtri3111996** [reply]
Hello @diegoteran, thanks for your reply, I updated to the newest version and it worked for me.

**Comment by 53RG1005** [comment]
Same, 3.1.1 not solve this problem
